lib/spanner/command/reqrep.ex

```diff
@@ -57,12 +48,10 @@ defmodule Spanner.Command.Request do
 
   defp read_config(request, config_path) do
     [bundle, _cmd] = String.split(request.command, ":")
```

Comment:
This will break with Slack emoji-named commands:

```elixir
iex(6)> [_bundle, cmd] = String.split("foo::thumbs_up:", ":")
** (MatchError) no match of right hand side value: ["foo", "", "thumbs_up", ""]
```

Use `parts: 2` to get the desired behavior:

```elixir
iex(6)> [_bundle, cmd] = String.split("foo::thumbs_up:", ":", parts: 2)
["foo", ":thumbs_up:"]
iex(7)> [_bundle, cmd] = String.split("foo:bar", ":", parts: 2)
["foo", "bar"]
```